doc/man3/X509_get_extension_flags.pod

```diff
@@ -86,8 +86,8 @@ ASN1 object itself.
 
 =item B<EXFLAG_NO_FINGERPRINT>
 
-Failed to compute the internal SHA1 hash value of the certificate or CRL.
-This may be due to malloc failure or because no SHA1 implementation was found.
+Failed to compute the internal SHA.1 hash value of the certificate or CRL.
+This may be due to malloc failure or because no SHA.1 implementation was found.
```

Comment:
```suggestion
Failed to compute the internal SHA-1 hash value of the certificate or CRL.
This may be due to malloc failure or because no SHA-1 implementation was found.
```